include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -128,7 +128,7 @@ class SplineBuilder
     template <class Layout>
     void operator()(
             ddc::ChunkSpan<double, ddc::DiscreteDomain<bsplines_type>, Layout, MemorySpace> spline,
-            ddc::ChunkSpan<double, interpolation_domain_type, Layout, MemorySpace> vals,
+            ddc::ChunkSpan<double const, interpolation_domain_type, Layout, MemorySpace> vals,
```

Comment:
```suggestion
            ddc::ChunkSpan<double, interpolation_domain_type, Layout, MemorySpace> vals,
```

Reply:
I revert this because I will remove spline_builder.hpp in next MR and will address all the issues related to it at this moment

Reply:
Well, the work was done so I do not see the benefit of reverting it to a worse state.